tools/build-tools/README.md

```diff
@@ -13,30 +19,66 @@ In this directory:
 ```sh
 npm i
 npm run build
-npm link
+```
+
+## Running these tools (Command Line)
+
+This package produces several binaries, see `bin` in package.json.
+
+This repo is normally build using the version of this package referenced by the root package.json file.
+To run that version, just do one of:
+- `npm i` in the root, and use the npm scripts that call it (ex: `build:fast`)
+- globally install that specific version of `@fluidframework/build-tools` and call its binaries directly (ex `fluid-build`)
+
+There are several also ways to use the version of `@fluidframework/build-tools` from within the repo.
```

Comment:
```suggestion
There are several also ways to use the local version of `@fluidframework/build-tools` from within the repo.
```
nit: as is reads a bit ambiguously as to what version this is actually referring to